clientv3/integration/dial_test.go

```diff
@@ -173,3 +260,52 @@ func TestDialForeignEndpoint(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// copyTLSFiles clones certs files to temp directory.
+func copyTLSFiles(ti transport.TLSInfo) (transport.TLSInfo, error) {
```

Comment:
`func copyTLSFiles(ti transport.TLSInfo, dst string) (transport.TLSInfo, error)` instead of bothering with tempdirs here; can be passed in as an argument